pandas/_testing/asserters.py

```diff
@@ -303,10 +303,10 @@ def assert_index_equal(
 
     Examples
     --------
-    >>> from pandas.testing import assert_index_equal
+    >>> import pandas as pd
```

Comment:
can you use the standard convntion e.g.

```
from pandas import testing as tm
```

Reply:
Just to clarify, do you mean using the standard convention on all modified import statements or on this specific line?

Reply:
on all

Reply:
Sorry, I am a first-time contributor, but isn't what you're mentioning the same as the original import statements, for example,  `from pandas.io.common import urlopen`?

Reply:
no there is a specific convention for testing routines. *only* those that are note user facing testing should use the fully qualified paths

Reply:
I updated import statements for user-facing testing files and put back fully qualified paths for others. Let me know if that'll work or if more updates are needed. Thank you. I also followed the following line in the contribution guidelines docs.
> Import required libraries (except numpy and pandas)